Tests/js/views/assets/ez-dashboardblocksview-tests.js

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblocksview-tests', function (Y) {
+    var renderTest,
+        forwardActiveTest,
+        addBlockTest,
+        removeBlockTest,
+        VIEW_CONFIG = {container: '.container'},
+        SELECTOR_CONTENT = '.dashboard-content',
+        View = Y.eZ.DashboardBlocksView;
+
+    Y.eZ.DashboardBlockBaseView = Y.Base.create('dashboardBlockBaseView', Y.View, [], {}, {
+        ATTRS: {
+           priority: {},
+            identifier: {}
+        }
+    });
+
+    renderTest = new Y.Test.Case({
+        name: 'eZ Dashboard Blocks View render test',
+
+        setUp: function () {
+            this.view = new View(VIEW_CONFIG);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        'Should render view correctly without blocks inside': function () {
+            var templateCalled = false,
+                origTpl = this.view.template;
+
+            this.view.template = function () {
+                templateCalled = true;
+
+                return origTpl.apply(this, arguments);
+            };
+
+            this.view.render();
+
+            Y.Assert.isTrue(templateCalled, 'The template should have been used to render view');
+        }
+    });
+
+    addBlockTest = new Y.Test.Case({
+        name: 'eZ Dashboard Blocks View add block tests',
```

Comment:
There should be also the test for testing whether the it doesn't render blocks with the same identifiers. So 4 tests. All of them are done.